src/runtime/data/kata-collect-data.sh

```diff
@@ -821,6 +836,10 @@ show_details()
 
 main()
 {
+	if [ "$1" = "show-image-details" ]; then
```

Comment:
Those changes belong to the next commit (https://github.com/kata-containers/kata-containers/pull/4736/commits/0678452283463731de2cdb3ca8256c895c318a86).